.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Documentation
+about: Report a problem with the documentation
```

Comment:
I'd be tempted to change this to "Give feedback, make a suggestion/request, or report a problem with the documentation". I realise it's a bit long, but feedback on the docs might be a bit broader than just reporting a problem with them.